app/controllers/organisations_controller.rb

```diff
@@ -1,8 +1,17 @@
 class OrganisationsController < ApplicationController
+  rescue_from "Pagy::OverflowError" do |exception|
```

Comment:
linter sez ....

```
app/controllers/organisations_controller.rb:2:41: W: Lint/UnusedBlockArgument: Unused block argument - exception. You can omit the argument if you don't care about it. (https://rubystyle.guide#underscore-unused-vars)
  rescue_from "Pagy::OverflowError" do |exception|
                                        ^^^^^^^^^
```

Would it be worth setting the `rescue_responses` for this exception like [we do in ttapi](https://github.com/DFE-Digital/teacher-training-api/blob/master/config/application.rb#L38)?

Reply:
Yep. Good shout. I've got another PR brewing with another exception that needs handling so I'll add it in there.